Framework/Kernel/test/TimeSeriesPropertyTest.h

```diff
@@ -581,6 +598,112 @@ class TimeSeriesPropertyTest : public CxxTest::TestSuite {
     TS_ASSERT_EQUALS(*tsp_result, *tsp_expected);
   }
 
+  void test_single_value_roi_mean() {
+    DateAndTime firstLogTime("2007-11-30T16:17:00");
+    TimeSeriesProperty<double> *prop = new TimeSeriesProperty<double>("doubleProp");
```

Comment:
Variable `prop` needs to be deleted at the end.  Or we can use a smart pointer, like here:  [https://github.com/mantidproject/mantid/blob/ewm1355_mean_math/Framework/Kernel/test/TimeSeriesPropertyTest.h#L464-L465](url)